release/create-release-branch.sh

```diff
@@ -16,12 +15,15 @@ RELEASE_REGEX="^[0-9][0-9]\.([1-9]|[1][0-2])$"
 
 update_products() {
   if [ -d "$BASE_DIR/$DOCKER_IMAGES_REPO" ]; then
+    echo "Directory exists. Switching to ${RELEASE_BRANCH} branch and Updating..."
     cd "$BASE_DIR/$DOCKER_IMAGES_REPO"
-    git pull && git switch "${RELEASE_BRANCH}"
+    git pull && git switch "${RELEASE_BRANCH}" # Switch to local branch (remote doesn't yet exist)
```

Comment:
question: Should we also try to create the branch (same as below) if it doesn't exist?